test/end-to-end.js

```diff
@@ -983,6 +983,56 @@ describe('end-to-end', function() {
   });
 });
 
+it('includes sensitive error details in development mode', function(done) {
```

Comment:
These two tests should live inside `describe('end-to-end')` block, since they are relying on SANDBOX being initialised with an app.